sentry-tracing/src/converters.rs

```diff
@@ -104,8 +147,14 @@ impl Visit for FieldVisitor {
 }
 
 /// Creates a [`Breadcrumb`] from a given [`tracing_core::Event`]
-pub fn breadcrumb_from_event(event: &tracing_core::Event) -> Breadcrumb {
```

Comment:
This is a breaking change anyway because this fn signature is gaining a new parameter. So might as well just go directly with an `Option`, as in `impl Into` is a bit heavy handed.

Reply:
Since I don't need it anyway, I can remove this new argument from this particular `breadcrumb_from_event`. I put it here before only to make the uniform-looking signatures.

Reply:
Done, restored the initial `breadcrumb_from_event` signature.